server/config/config.go

```diff
@@ -200,6 +204,8 @@ func (man Manager) LoadConfig() KolideConfig {
 			TLSCA:         man.getConfigString("mysql.tls_ca"),
 			TLSServerName: man.getConfigString("mysql.tls_server_name"),
 			TLSConfig:     man.getConfigString("mysql.tls_config"),
+			MaxOpenConns:  man.getConfigString("mysql.max_open_conns"),
```

Comment:
`getConfigInt`

Reply:
Fixed. Wasn't sure about that one so I followed the pattern.